src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/ISolutionExtensions.cs

```diff
@@ -46,6 +46,9 @@ public static Project GetRequiredProject(this Solution solution, ProjectId proje
 
         public static Document GetRequiredDocument(this Solution solution, DocumentId documentId)
         {
+            if (documentId is null)
+                throw new ArgumentNullException(nameof(documentId));
```

Comment:
needed to see if we're crashing because we passed in null here errantly, or if we passed in a real doucment id and didn't find it.